fs/tarfs.py

```diff
@@ -280,9 +281,19 @@ def _directory(self):
         if self._directory_cache is None:
             _decode = self._decode
             _directory = ((_decode(info.name).strip("/"), info) for info in self._tar)
```

Comment:
When I was debugging the linked issue, I found the `_directory` var-name very confusing. Perhaps `_directory_entries` or something would be better?

Reply:
Good point